doc/sailfish-webview-qmozenginesettings.h

```diff
@@ -0,0 +1,79 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <QObject>
+
+// Used only for documentation purposes
+class QMozEngineSettings : public QObject {
```

Comment:
This doesn't indeed look so nice but I'm ok to live with this at least for now.

Reply:
Let me merge the `QtMozEngineSettings` into the `WebEngineSettings` docs. It should just be a massive cut `n paste task.

Reply:
By which I mean "a (massive cut 'n paste) task" and not "a massive (cut `n paste task)". Natural language would be better with brackets.

Reply:
I've removed the QMozEngineSettings docs and merged them into the WebEngineSettings. If you approve all squash the commits.

I can do this for WebEngine too?